functions/src/judge.ts

```diff
@@ -325,9 +332,46 @@ export async function get_verdict(req: Request, res: Response) {
           new_object.pending = pending
           new_object.memory = memory
 
-          updateDoc(doc(db, 'Submissions', submission_id), new_object)
+          let verdictUpdate = {}
+          if (!pending) {
+            switch (verdict) {
+              case 3: {
+                verdictUpdate = { problemsAccepted: increment(1) }
```

Comment:
Could we rename this to something like "submissionsAccepted," to reflect the fact that we're counting up **all** submission verdicts, and not just each problem's status?

Same thing with the others.